### DISCUSSION TITLE
How can I get whether the point is occupied by a vehicle

### DISCUSSION THREAD

**Discussion by rogercc116** [openTCS/opentcs#93] [Q&A]
I know that the canSendNextCommand method is currently used to control whether the command is issued, but I have not found a way to judge whether the point is in the blocking area.Can you guide me? Thank you

**Comment by swltr** [comment]
With "in the blocking area", I assume you mean a `Block`. Is that correct? That kind of decision belongs to traffic management, i.e. the `Scheduler` implementation.

Can you explain a bit more what you are trying to achieve?
